pom.xml

```diff
@@ -37,6 +37,11 @@
             <version>RELEASE</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
```

Comment:
what is this dependency used for?

Reply:
mvn install failed earlier when i took it out, but now it works, i removed it!